packages/app-page-builder/src/admin/components/Table/Header/Search/Search.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import SearchUI from "@webiny/app-admin/components/SearchUI";
+
+import { SearchWrapper } from "./styled";
+
+interface SearchProps {
+    value: string;
+    onChange: (value: string) => void;
+}
+
+export const Search: React.VFC<SearchProps> = ({ value, onChange }) => {
+    return (
+        <SearchWrapper>
+            <SearchUI value={value} onChange={onChange} />
+        </SearchWrapper>
+    );
+};
```

Comment:
This component is already copied twice in this PR. Can we make this component part of the `app-aco` package, and just use it from there?